### PR TITLE
botmeta: exoscale migrated to ngine_io.exoscale

### PR DESCRIPTION
**Pull request by resmo**
##### SUMMARY
exoscale related modules migrated to dedicated collection. 

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME
exoscale

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Review by gundalow — APPROVED** [review]

File: lib/ansible/config/ansible_builtin_runtime.yml [hunk 1]
```diff
@@ -1653,9 +1653,9 @@ plugin_routing:
     dnsmadeeasy:
```
Comment (gundalow) [inline lib/ansible/config/ansible_builtin_runtime.yml:1653]: Missing module_utils from here

**Review by gundalow — CHANGES_REQUESTED** [review]

File: lib/ansible/config/ansible_builtin_runtime.yml (hunk 1 above)
Comment (felixfontein) [reply lib/ansible/config/ansible_builtin_runtime.yml:1653]: Also doc_fragments.

**Comment by resmo** [comment]
ready_for_review

File: lib/ansible/config/ansible_builtin_runtime.yml [hunk 2]
```diff
@@ -8333,7 +8333,7 @@ plugin_routing:
     enos:
       redirect: community.network.enos
     exoscale:
-      redirect: community.general.exoscale
+      redirect: ngine.io.exoscale.exoscale
```
Comment (felixfontein) [inline lib/ansible/config/ansible_builtin_runtime.yml]:
```suggestion
      redirect: ngine_io.exoscale.exoscale
```

**Comment by resmo** [comment]
@felixfontein good catch. Thanks

**Review by felixfontein — APPROVED** [review]

**Review by gundalow — APPROVED** [review]

**Comment by gundalow** [comment]
rebuild_merge

**Pull request merged**